# 位置： 0-255
int32 position

# 速度：0-255
int32 speed

# 力矩：0-255
int32 torque